src/commands/deploy/utils.ts

```diff
@@ -255,7 +258,7 @@ export async function enableAuthentication(
             {
                 enabled: true,
                 databaseUrl: databaseUri,
-                databaseType: authDatabase.type,
+                databaseType: authDatabase.type as AuthenticationDatabaseType,
```

Comment:
Why do you need the type cast? Can't we make the type system understand this type? Zod should attest that this type is really a auth db type.

Reply:
Yes, you are correct. I have updated the types to correctly enforce `AuthenticationDatabaseType` without any cast.
Thanks